python/tvm/target/target.py

```diff
@@ -284,6 +284,7 @@ def intel_graphics(model="unknown", options=None):
     "stm32f746xx": ["-mcpu=cortex-m7", "-march=armv7e-m"],
     "nrf5340dk": ["-mcpu=cortex-m33"],
     "mps2_an521": ["-mcpu=cortex-m33"],
+    "stm32l4r5zi": ["-mcpu=cortex-m4"],
```

Comment:
```suggestion
    "mps2_an521": ["-mcpu=cortex-m33"],
    "nrf5340dk": ["-mcpu=cortex-m33"],
    "stm32f746xx": ["-mcpu=cortex-m7", "-march=armv7e-m"],
    "stm32l4r5zi": ["-mcpu=cortex-m4"],
```

Again, this is a massive nit, but I suggest we keep the boards, for our own sanity, alphabetically here, where possible.